CPS-????/README.md

```diff
@@ -0,0 +1,96 @@
+---
+CPS: ?
+Title: Light Client Protocols for Cardano
+Status: Open
+Category: Tools
+Authors:
+    - Sebastian Bode <sebastian.bode@cardanofoundation.org>
+Proposed Solutions: []
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/?
+Created: 2024-11-18
+License: CC-BY-4.0
+---
+
+<!-- Existing categories:
+
+- Meta     | For meta-CIPs which typically serves another category or group of categories.
+- Wallets  | For standardisation across wallets (hardware, full-node or light).
+- Tokens   | About tokens (fungible or non-fungible) and minting policies in general.
+- Metadata | For proposals around metadata (on-chain or off-chain).
+- Tools    | A broad category for ecosystem tools not falling into any other category.
+- Plutus   | Changes or additions to Plutus
+- Ledger   | For proposals regarding the Cardano ledger (including Reward Sharing Schemes)
+- Catalyst | For proposals affecting Project Catalyst / the Jörmungandr project
+
+-->
+
```

Comment:
```suggestion
```
(remove comment scaffolding)

It seems I've also forgotten to update, in the last CIP-0001 revision, the template files to include & define the `Consensus` and `Network` categories.  I'm leaving this as a "note to self" to fix these files ASAP:
- https://raw.githubusercontent.com/cardano-foundation/CIPs/refs/heads/master/.github/CIP-TEMPLATE.md
- https://raw.githubusercontent.com/cardano-foundation/CIPs/refs/heads/master/.github/CPS-TEMPLATE.md

Reply:
(will add `Consensus` & `Network` categories in forthcoming PR)